### PR TITLE
[SMALLFIX] Reenable secondary master

### PR DESCRIPTION
**Pull request by peisun1115**
It slows down tests by 41% (22min -> 31min).

### CODE REVIEW

**Comment by calvinjia** [comment]
@peisun1115 Since the slow down is nontrivial, is it possible to do this only for tests involving the secondary master?

**Comment by jsimsa** [comment]
What is the slowdown if your set the quiet window to 0?

File: minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java [hunk 1]
```diff
@@ -112,8 +112,7 @@ public WorkerNetAddress getWorkerAddress() {
   protected void startMasters() throws Exception {
     mMaster = LocalAlluxioMaster.create(mWorkDirectory);
     mMaster.start();
-    // TODO(peis): Reenable this. This is slowing down the tests.
-    // mMaster.startSecondary();
+    mMaster.startSecondary();
```
Comment (jsimsa) [inline minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java]: are we stopping the secondary master?

Comment (jsimsa) [inline minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java]: looking at the code, we are stopping secondary master in the stop method while we have separate start and startSecondary master methods ... any reason for this assymetry? if not, please fix that, thanks

**Comment by peisun1115** [comment]
@jsimsa It did not help much by further reducing the wait time to 0.

**Pull request closed without merging**